container-registry/container-analysis/src/main/java/com/example/containeranalysis/Samples.java

```diff
@@ -46,47 +48,56 @@
 import io.grpc.StatusRuntimeException;
 import java.io.IOException;
 import java.lang.InterruptedException;
+import java.util.concurrent.TimeUnit;
+// [END containeranalysis_imports_samples]
 
 /**
  * API usage samples
  */
 public class Samples {
 
-  // [START create_note]
+  // [START containeranalysis_create_note]
   /**
-   * Creates and returns a new vulnerability Note
-   * @param client The Grafeas client used to perform the API requests.
+   * Creates and returns a new Note
    * @param noteId A user-specified identifier for the Note.
    * @param projectId the GCP project the Note will be created under
    * @return the newly created Note object
+   * @throws IOException on errors creating the Grafeas client
+   * @throws InterruptedException on errors shutting down the Grafeas client
    */
-  public static Note createNote(GrafeasV1Beta1Client client, String noteId, String projectId) {
+  public static Note createNote(String noteId, String projectId) 
+      throws IOException, InterruptedException {
+    final String projectName = ProjectName.format(projectId);
+
     Note.Builder noteBuilder = Note.newBuilder();
     Vulnerability.Builder vulBuilder = Vulnerability.newBuilder();
     // Details about the your vulnerability can be added here
     // Example: vulBuilder.setSeverity(Severity.CRITICAL);
     noteBuilder.setVulnerability(vulBuilder);
     Note newNote = noteBuilder.build();
 
-    final String projectName = ProjectName.format(projectId);
-    return client.createNote(projectName, noteId, newNote);
+    GrafeasV1Beta1Client client = GrafeasV1Beta1Client.create();
+    Note result = client.createNote(projectName, noteId, newNote);
+    client.shutdownNow();
```

Comment:
Why are we doing this here?  Is this how all users would use the createNote api?  Or is this just a convenience for us in this moment? (if the just for this sample, then maybe we should put it somewhere else or at least put a comment about it here)